subcommands/waves/init.go

```diff
@@ -145,6 +150,31 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(api.FactoryCreateWave(factory, &wave), "Failed to create a wave")
 	}
+
+	if !hasStaticDelta(new_targets) {
+
+		fmt.Print(`
+WARNING: You created a wave for a target version without static deltas.
+
+We recommend that you generate static deltas for your production targets to optimize
+the size of the update. Please, consider generating a static delta for targets using:
+$ fioctl targets static-deltas.
+
+You can then cancel this wave and create a new one for a target with a static delta.
+`)
+
+	}
+}
+
+func hasStaticDelta(new_targets tuf.Files) bool {
+	for _, file := range new_targets {
+		custom, err := api.TargetCustom(file)
+		subcommands.DieNotNil(err)
+		if custom.DeltaStats != nil {
```

Comment:
I think we should return `true` only if delta is present for each target. So, maybe 
```
if custom.DeltaStats == nil {
  return false;
}
```

Reply:
Ops. Thx